example/spec/parallel/testAPI/bulkOperationsSuccessSpec.js

```diff
@@ -0,0 +1,293 @@
+                      path: sourcePath,
+                    },
+                  ],
+                },
+              ],
+            },
+          }
+        );
+
+        // Find the execution ARN
+        const firstIngestGranuleExecutionArn = await findExecutionArn(
+          prefix,
+          (execution) => {
+            const executionId = get(execution, 'originalPayload.testExecutionId');
+            return executionId === ingestGranuleRule.payload.testExecutionId;
+          },
+          { timestamp__from: ingestTime },
+          { timeout: 15 }
+        );
+
+        // Wait for the execution to be completed
+        await getExecutionWithStatus({
+          prefix,
+          arn: firstIngestGranuleExecutionArn,
+          status: 'completed',
+          timeout: 60,
+        });
+
+        // Wait for the granule to be fully ingested
+        ingestedGranule = await getGranuleWithStatus({ prefix, granuleId, status: 'completed' });
+
+        scheduleQueueUrl = await getQueueUrlByName(`${config.stackName}-backgroundProcessing`);
+        bulkRequestTime = Date.now() - 1000 * 30;
+        postBulkGranulesResponse = await granules.bulkGranules({
+          prefix,
+          body: {
+            ids: [granuleId],
+            workflowName: 'HelloWorldWorkflow',
+            queueUrl: scheduleQueueUrl,
+          },
+        });
+        postBulkOperationsBody = JSON.parse(postBulkGranulesResponse.body);
+
+        // Query the AsyncOperation API to get the task ARN
+        const getAsyncOperationResponse = await apiTestUtils.getAsyncOperation({
+          prefix,
+          id: postBulkOperationsBody.id,
+        });
+        ({ taskArn } = JSON.parse(getAsyncOperationResponse.body));
+        beforeAllSucceeded = true;
+      } catch (error) {
+        console.log(error);
+      }
+    });
+
+    afterAll(async () => {
+      // Must delete rules before deleting associated collection and provider
+      await deleteRule({ prefix, ruleName: get(ingestGranuleRule, 'name') });
+
+      await pAll(
+        [
+          () => deleteProvider({ prefix, providerId: get(provider, 'id') }),
+          () => deleteCollection({
+            prefix,
+            collectionName: get(collection, 'name'),
+            collectionVersion: get(collection, 'version'),
+          }),
+          () => granules.deleteGranule({ prefix, granuleId }),
+        ],
+        { stopOnError: false }
+      ).catch(console.error);
+    });
+
+    it('ingested granule is archived', () => {
+      expect(beforeAllSucceeded).toBeTrue();
+      expect(ingestedGranule).toBeTruthy();
+    });
+
+    it('returns a status code of 202', () => {
+      expect(beforeAllSucceeded).toBeTrue();
+      expect(postBulkGranulesResponse.statusCode).toEqual(202);
+    });
+
+    it('returns an Async Operation Id', () => {
+      expect(beforeAllSucceeded).toBeTrue();
+      expect(postBulkOperationsBody.id).toMatch(/[\da-f]{8}(?:-[\da-f]{4}){3}-[\da-f]{12}/);
```

Comment:
It looks like we're using this regex in a couple other tests (e.g. `bulkdDeleteFailureSpec`). Could we put it in a test utils somewhere?